sdk/src/main/java/com/fingerprint/model/WebhookFactoryReset.java

```diff
@@ -64,11 +64,11 @@ public WebhookFactoryReset timestamp(Long timestamp) {
   }
 
    /**
-   * This field is just another representation of the value in the &#x60;time&#x60; field. The time of the most recent factory reset that happened on the **mobile device** is expressed as Unix epoch time. 
+   * This field is just another representation of the value in the &#x60;time&#x60; field. The time of the most recent factory reset that happened on the **mobile device** is expressed as Unix epoch time.        
    * @return timestamp
   **/
   @jakarta.annotation.Nullable
-  @Schema(description = "This field is just another representation of the value in the `time` field. The time of the most recent factory reset that happened on the **mobile device** is expressed as Unix epoch time. ")
+  @Schema(description = "This field is just another representation of the value in the `time` field. The time of the most recent factory reset that happened on the **mobile device** is expressed as Unix epoch time.        ")
```

Comment:
weird spacing. I didn't see this [in Go PR](https://github.com/fingerprintjs/fingerprint-pro-server-api-go-sdk/pull/114/files) cc @ilfa

Reply:
It is because I updated schema manually last time and we have an issue with weird spaces when we do it 100% automatically =(